calm/spectral/calm-rules.yaml

```diff
@@ -0,0 +1,66 @@
+extends: spectral:oas
+functions: []  # We'll implement custom functions in the next step
+
+rules:
+  calm-schema-version:
+    description: 'Ensures correct JSON Schema version'
+    message: 'Must use JSON Schema Draft 2020-12'
+    severity: error
+    given: $.['$schema']
```

Comment:
I don't think this will raise an error if `$schema` is missing, I think it should be:
```
rules:
  calm-schema-version:
    description: 'Ensures correct JSON Schema version'
    message: 'Must use JSON Schema Draft 2020-12'
    severity: error
    given: $
    then:
      field: $schema 
      function: pattern
      functionOptions:
        match: https://json-schema.org/draft/2020-12/schema
```